applications/addons/settings/bootstrap.php

```diff
@@ -0,0 +1,3 @@
+<?php if (!defined('APPLICATION')) exit();
+
+require_once(PATH_APPLICATIONS . DS . 'addons' . DS . 'library' . DS . 'functions.render.php');
```

Comment:
I would generally expect to see a `helper_functions.php` in the views folder included from the views that need it rather than a global functions.render, especially when it's the only one and used in a limited context.